test/functional/interface_rpc.py

```diff
@@ -18,6 +21,16 @@ def expect_http_status(expected_http_status, expected_rpc_code,
         assert_equal(exc.error["code"], expected_rpc_code)
         assert_equal(exc.http_status, expected_http_status)
 
+
+def test_work_queue_getblock(node, got_exceeded_error):
```

Comment:
nit: this is called `queue_getblock` but queues a `getrpcinfo` ? Maybe `queue_rpc_call`